src/migration/migrator.ts

```diff
@@ -509,11 +509,17 @@ export class Migrator {
     const executedMigrations = await db
       .withPlugin(this.#schemaPlugin)
       .selectFrom(this.#migrationTable)
-      .select('name')
-      .orderBy(['timestamp', 'name'])
+      .select(['name', 'timestamp'])
       .execute()
 
-    return executedMigrations.map((it) => it.name)
+    return executedMigrations
+      .sort((a, b) => {
+        if (a.timestamp === b.timestamp) {
+          return a.name < b.name ? -1 : 1
```

Comment:
change to [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare).